source/deployment-paradigms/multi-region.txt

```diff
@@ -62,6 +62,133 @@ multiple geographies, ensures the highest level of availability with a single
 provider. If your application requires the very highest level of availability 
 and lowest latency, consider a :ref:`arch-center-paradigms-multi-cloud`.
 
+Multi-Region Deployment Use Cases
+---------------------------------
+
+The following use cases highlight a high-criticality application ("Tier 1") 
+and a less-critical application, but one that still warrants a 
+multi-region deployment ("Tier 2").
+
+.. _arch-center-multi-region-tier-1:
+
+Tier 1 Applications
+~~~~~~~~~~~~~~~~~~~
+
+Tier 1 applications require both high availability and low-latency. 
+To achieve this, we recommend an architecture that consists of a minimum of 5 
+nodes spread over at least 3 regions. This provides high availability in at least 
+2 regions. If a node goes down, the traffic can be served from the same 
+region to ensure the same latency on the primary region. 
+
+.. figure:: /includes/images/multi-region-tier-1.png
+   :figwidth: 750px
+   :alt: An image showing three types of multi-region deployments
+
+Notes and Considerations
+````````````````````````
+
+- Your application should also be deployed in each region to further reduce  
+  latency.
+
+- You should use Privatelink for connecting to the cluster, and VPC peering 
```

Comment:
Link to Private Link and VPC peering docs.

Reply:
Also, I think generically these are called "private endpoints"? Privatelink is AWS and Private Link is Azure and GCP calls it something else?